bin/cloudgov/login

```diff
@@ -1,17 +1,28 @@
 #!/bin/bash
 set -e
 
-# we might be running in circleci
-if [ -f /home/circleci/project/env.local ]; then
-  . /home/circleci/project/env.local
-fi
-# we might be running from a local dev machine
-SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
-if [ -f $SCRIPT_DIR/env.local ]; then
-  . $SCRIPT_DIR/env.local
-fi
-if [ -f ./env.local ]; then
-  . ./env.local
+usage="
+$0: Logs in to cloud.gov
+
+Usage:
+   $0 -h
+   $0 USER PASS
+   $0 USER PASS SPACE
```

Comment:
This is fascinating: I've been running "bin/cloudgov/login --sso" because I either saw that somewhere, or mixed it up with the cf login command, and now I see that the only reason it works is that I'm giving the script a number of commands it doesn't like, and --sso is the fallback. 

I don't know whether there's anything to change here. I'm thinking more and more that we should just use --sso and not provision service credentials. Thoughts?

Reply:
we could, I suppose, but having service credentials is convinent and lets us give more limited access to other folks